### ISSUE TITLE
i can't game on it no game detects that it is a monitor

### ISSUE THREAD

**Issue by Space-Clustercompany** [itsmikethetech/Virtual-Display-Driver#89]

**Comment by itsmikethetech** [comment]
Is it showing up and enabled in display settings?

**Comment by Space-Clustercompany** [comment]
Yes i can use it for desktop stuff but when i want to full screen a game on it it doesn't let me and in the games settings it doesn't show the monitor exists

**Comment by anderse77** [comment]
I cannot either use it because it does not show up in nvidia control panel

**Comment by bud3699** [comment]
This is most likely how some games iterate through displays. Not much we can do about this. I'm marking as wont fix for now, although its more like "cant fix". I'll do some more research into it, potentially if we were to get this working we may have to re-write the entire driver by emulating a usb device as a display.
